src/DynamoCoreWpf/ViewModels/Search/NodeAutoCompleteSearchViewModel.cs

```diff
@@ -85,9 +85,14 @@ internal IEnumerable<NodeSearchElement> GetMatchingSearchElements()
         {
             var elements = new List<NodeSearchElement>();
             var inputPortType = PortViewModel.PortModel.GetInputPortType();
+
+            //List of nodes that are skipped temporarily, and will display list of default suggestions instead.
+            var skippedInputPorts = new List<string>() { "var", "object", "string", "bool", "int", "double" };
+          
+            if (inputPortType == null || (skippedInputPorts.Any(s => s == inputPortType.ToString()))) return elements;
```

Comment:
@QilongTang shouldn't we be returning the default search elements in this case?

Reply:
@aparajit-pratap Yes, it depends on where you would like to implement. I think @zeusongit decides to put it inside of the API itself, we can also implement as part of PopulateAutoCompleteCandidates(). That's why I was asking your preference if you have any

Reply:
Yes, by returning `null` here, it is handled in the `PopulateAutoCompleteCandidates` that fills the default values